CI failing on picolibc

```
picolibc Config
picolibc
  picolibc 0.0.post21348 (0, 0, 21348)
  Tools: 0.0.post108 (0, 0, 108)
   Data: 0.0.post21240 (0, 0, 21240)
  Updating repo  litex-hub/pythondata-software-picolibc
  404 {"message": "Not Found", "documentation_url": "https://docs.github.com/rest/reference/repos#update-a-repository"}
  Cloning into 'repos/pythondata-software-picolibc'...
  Submodule 'pythondata_software_picolibc/data' (https://github.com/picolibc/picolibc) registered for path 'pythondata_software_picolibc/data'
  Cloning into '/home/runner/work/pythondata-auto/pythondata-auto/repos/pythondata-software-picolibc/pythondata_software_picolibc/data'...
  Submodule path 'pythondata_software_picolibc/data': checked out '53567cf82b92f1bd777ed57111232a110666b4df'
  
  Updating: pythondata-software-picolibc
  ---------------------------------------------------------------------------
    Updating repos/pythondata-software-picolibc                           from /home/runner/work/pythondata-auto/pythondata-auto/templates
    Creating repos/pythondata-software-picolibc/pythondata_software_picolibc from /home/runner/work/pythondata-auto/pythondata-auto/templates/__py__
    Creating repos/pythondata-software-picolibc/.github                   from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github
     Copying repos/pythondata-software-picolibc/Makefile                  from /home/runner/work/pythondata-auto/pythondata-auto/templates/Makefile
   Rendering repos/pythondata-software-picolibc/MANIFEST.in               from /home/runner/work/pythondata-auto/pythondata-auto/templates/MANIFEST.in.jinja
   Rendering repos/pythondata-software-picolibc/README.md                 from /home/runner/work/pythondata-auto/pythondata-auto/templates/README.md.jinja
     Copying repos/pythondata-software-picolibc/requirements.txt          from /home/runner/work/pythondata-auto/pythondata-auto/templates/requirements.txt
   Rendering repos/pythondata-software-picolibc/setup.py                  from /home/runner/work/pythondata-auto/pythondata-auto/templates/setup.py.jinja
   Rendering repos/pythondata-software-picolibc/test.py                   from /home/runner/work/pythondata-auto/pythondata-auto/templates/test.py.jinja
     Copying repos/pythondata-software-picolibc/.gitattributes            from /home/runner/work/pythondata-auto/pythondata-auto/templates/.gitattributes
     Copying repos/pythondata-software-picolibc/.gitignore                from /home/runner/work/pythondata-auto/pythondata-auto/templates/.gitignore
    Updating repos/pythondata-software-picolibc/pythondata_software_picolibc from /home/runner/work/pythondata-auto/pythondata-auto/templates/__py__
   Rendering repos/pythondata-software-picolibc/pythondata_software_picolibc/__init__.py from /home/runner/work/pythondata-auto/pythondata-auto/templates/__py__/__init__.py.jinja
    Updating repos/pythondata-software-picolibc/.github                   from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github
    Creating repos/pythondata-software-picolibc/.github/workflows         from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github/workflows
    Updating repos/pythondata-software-picolibc/.github/workflows         from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github/workflows
     Copying repos/pythondata-software-picolibc/.github/workflows/publish-to-pypi.yml from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github/workflows/publish-to-pypi.yml
  ---------------------------------------------------------------------------
  [master 5285816] Updating pythondata-software-picolibc to 0.0.post21348
   1 file changed, 20 insertions(+), 15 deletions(-)
  git submodule update --remote --merge
  Updating 53567cf82..4b4a69f42
  Fast-forward
   meson.build | 1 -
   1 file changed, 1 deletion(-)
  Submodule path 'pythondata_software_picolibc/data': merged in '4b4a69f42f2e6fa5d6e44607adefe10f38d1b51a'
  Traceback (most recent call last):
    File "./update.py", line 588, in <module>
      sys.exit(main(sys.argv[0], sys.argv[1:]))
    File "./update.py", line 567, in main
      update(m)
    File "./update.py", line 405, in update
      f.flush()
    File "/usr/lib/python3.6/tempfile.py", line 624, in func_wrapper
      return func(*args, **kwargs)
  ValueError: flush of closed file
  Error: Process completed with exit code 1.
```

--- a/update.py
+++ b/update.py
@@ -402,7 +402,7 @@ Bump {dir} submodule to {data_git_hash}
 
 Updated using {tool_version} from https://github.com/litex-hub/litex-data-auto
 """.format(**module_data).encode('utf-8'))
-                f.flush()
+                    f.flush()
                 subprocess_check_call(['git', 'commit', '-F', f.name], cwd=repo_dir)
 
         else:

